synapseclient/client.py

```diff
@@ -3853,47 +3853,104 @@ def _find_teams_for_principal(self, principal_id: str) -> typing.Iterator[Team]:
         for result in self._GET_paginated(f"/user/{principal_id}/team"):
             yield Team(**result)
 
-    @tracer.start_as_current_span("Synapse::getTeam")
-    def getTeam(self, id):
+    def create_team(
```

Comment:
What was your rationale in creating this function?  Right now you can create a team by doing:

```
syn.store(Team(...))
```